src/include/utils/first_follow.h

```diff
@@ -1,6 +1,7 @@
 #ifndef JUCC_FIRST_FOLLOW_H
 #define JUCC_FIRST_FOLLOW_H
 
+#include <algorithm>
```

Comment:
why is this header included?

Reply:
std::find

Reply:
include in .cpp file then .h doesn't require this.